index.js

```diff
@@ -0,0 +1,105 @@
+
+let url="http://localhost:3000/school";
+let school="";
```

Comment:
if you don't know what the variable is going to be you can simple define it by `let school;`